CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ include(FetchContent)
 
 FetchContent_Declare(DirectXMath
                      GIT_REPOSITORY https://github.com/NcStudios/DirectXMath.git
-                     GIT_TAG        21b8256207a68d413c517f97391f8354e5211692 # origin/release
+                     GIT_TAG        v3.1.6+nc.1
```

Comment:
Commit hashes don't play nice with shallow cloning, but tags are ok. The '+nc.1' on the tag here is semver conforming 'build metadata' to disambiguate from upstream.